### PR TITLE
[Testing:TAGrading] TA Rubric Attachments Access

### PR DESCRIPTION
**Pull request by manzil-infinity180**
Issue #9598 

Cypress test for uploading the attachment and deleting  it by TA and also checking is that student have access to that path or not - 
Path 1 - /courses/s24/sample/download
Path 2 - courses/s24/sample/gradeable/grading_homework/grading/details
as done by old pr but the test is in selenium 

@IDzyre i want to ask , should i add some more test to it or it is enough, if any let me know what ??

### CODE REVIEW

**Review by williamjallen — APPROVED** [review]
Seems reasonable to me.  @IDzyre feel free to merge if you approve since this is a test-only change.

File: site/cypress/e2e/Cypress-Gradeable/test_rubric_access.spec.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+describe('Rubric Access Test', () => {
+    it('test sample file exists student', () => {
+        cy.login('aphacker');
+        cy.visit('/courses/s24/sample/download');
+        cy.get('.content').should('contain', "You don't have access to this page.");
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'details']);
+        cy.get('#error-0').should('exist').and('contain', 'You do not have permission to grade Grading Homework');
```
Comment (IDzyre) [inline site/cypress/e2e/Cypress-Gradeable/test_rubric_access.spec.js:7]: you don't need the should('exist) here, and can make it just should('contain'... same as line 15, 18, and 20.

**Review by IDzyre — CHANGES_REQUESTED** [review]

File: site/cypress/e2e/Cypress-Gradeable/test_rubric_access.spec.js (hunk 1 above)
Comment (IDzyre) [reply site/cypress/e2e/Cypress-Gradeable/test_rubric_access.spec.js:7]: I will approve and merge once this is changed.

Comment (manzil-infinity180) [reply site/cypress/e2e/Cypress-Gradeable/test_rubric_access.spec.js]: done ✅, i will take care of this from now thank u

**Review by IDzyre — APPROVED** [review]

**Pull request merged**